### PR TITLE
[Fix-10895][standalone] fix standalone-server view task log

### PR DESCRIPTION
**Pull request by liguotian**
* add logger server to StandaloneServer
* add logback-standalone.xml file for StandaloneServer
* add log.config param to dolphinscheduler-daemon.sh start func

### CODE REVIEW

**Comment by caishunfeng** [comment]
I had created a new branch `2.0.7-prepare` and set this pr base to it.

File: dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java [hunk 1]
```diff
@@ -40,7 +41,8 @@ public static void main(String[] args) throws Exception {
             MasterServer.class,
             WorkerServer.class,
             AlertServer.class,
-            PythonGatewayServer.class
+            PythonGatewayServer.class,
+            AdaptedLoggerServer.class
```
Comment (caishunfeng) [inline dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java:45]: It seems you can use `LoggerServer` directly.

Comment (liguotian) [reply dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java:45]: LoggerServer doesn't have any LifeCycle function.so i create a AdaptedLoggerServer to start and stop in spring context.

Comment (liguotian) [reply dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java:45]: Or should I add  @PostConstruct And  IStoppable to LoggerServer . If this is batter ,I will  modify it。

**Comment by liguotian** [comment]
> SonarCloud Quality Gate failed.    [Quality Gate failed](https://sonarcloud.io/dashboard?id=apache-dolphinscheduler&pullRequest=10912)
> 
> [Bug](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=BUG) [E](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=BUG) [52 Bugs](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=VULNERABILITY) [B](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=VULNERABILITY) [6 Vulnerabilities](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=SECURITY_HOTSPOT) [E](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=SECURITY_HOTSPOT) [11 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=CODE_SMELL) [1319 Code Smells](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=CODE_SMELL)
> 
> [](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=10912&metric=new_coverage&view=list) [19.2% Coverage](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=10912&metric=new_coverage&view=list) [](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=10912&metric=new_duplicated_lines_density&view=list) [4.6% Duplication](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=10912&metric=new_duplicated_lines_density&view=list)

it seems  SonarCloud  scan all code。what can i do with this @caishunfeng

File: dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java (hunk 1 above)
Comment (caishunfeng) [reply dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java:45]: > Or should I add @PostConstruct And IStoppable to LoggerServer . If this is batter ,I will modify it。

Yes, I think so.

**Comment by caishunfeng** [comment]
> > SonarCloud Quality Gate failed.    [Quality Gate failed](https://sonarcloud.io/dashboard?id=apache-dolphinscheduler&pullRequest=10912)
> > [Bug](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=BUG) [E](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=BUG) [52 Bugs](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=VULNERABILITY) [B](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=VULNERABILITY) [6 Vulnerabilities](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=SECURITY_HOTSPOT) [E](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=SECURITY_HOTSPOT) [11 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=CODE_SMELL) [1319 Code Smells](https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&pullRequest=10912&resolved=false&types=CODE_SMELL)
> > [](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=10912&metric=new_coverage&view=list) [19.2% Coverage](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=10912&metric=new_coverage&view=list) [](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=10912&metric=new_duplicated_lines_density&view=list) [4.6% Duplication](https://sonarcloud.io/component_measures?id=apache-dolphinscheduler&pullRequest=10912&metric=new_duplicated_lines_density&view=list)
> 
> it seems SonarCloud scan all code。what can i do with this @caishunfeng

Yes, it seems scan all the code :joy:

**Comment by liguotian** [comment]
> 

i think it maybe 2.0.7-prepare is not exist in my fork repositories 。now I have synchronized the latest repositories.and reopen this PR

File: dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java (hunk 1 above)
Comment (liguotian) [reply dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java:45]: Ok, I get it.

**Pull request closed without merging**